rx888_source/src/main.cpp

```diff
@@ -52,10 +133,12 @@ class RX888SourceModule : public ModuleManager::Instance {
         handler.stream = &stream;
         sigpath::sourceManager.registerSource("RX888", &handler);
 
-        spdlog::info("RX888SourceModule '{0}': Instance created!", name);
+        spdlog::info("RX888SourceModule '{0}'-'{1}'-v{2}: Instance created!", name, sddc_get_hw_model_name(sddc), sddc_get_firmware(sddc));
```

Comment:
Will cause a segfault if a null value is returned, needs checking

Reply:
I added checking, but it isn't really needed/